eng/common/scripts/Detect-Api-Changes.ps1

```diff
@@ -61,8 +61,8 @@ function Should-Process-Package($pkgPath, $packageName)
     # Get package info from json file created before updating version to daily dev
     $pkgInfo = Get-Content $pkgPropPath | ConvertFrom-Json
     $packagePath = $pkgInfo.DirectoryPath
-    $modifiedFiles = git diff --name-only --relative $TargetBranch HEAD
-    $modifiedFiles = $modifiedFiles.Where({$_.startswith($packagePath)})
+    $gitDiffChanges = Join-Path $PSScriptRoot "git-diff-changes.ps1"
```

Comment:
Currently your script will exclude deleted files but in https://github.com/Azure/azure-sdk-tools/pull/2833/files#r817902643 it seems like @praveenkuttappan wants those as well.

Reply:
Oh I think that is what you are trying for with the FilterType.